pkg/pillar/types/volumetypes.go

```diff
@@ -411,3 +411,8 @@ func (status VolumeRefStatus) LogDelete(logBase *base.LogObject) {
 func (status VolumeRefStatus) LogKey() string {
 	return string(base.VolumeRefStatusLogType) + "-" + status.Key()
 }
+
+// Equals return true if VolumeRefStatus are equal
+func (status VolumeRefStatus) Equals(anotherStatus VolumeRefStatus) bool {
```

Comment:
The normal name for such functions is Equal without an 's'

Reply:
I moved it to caller